src/components/ProductList.jsx

```
@@ -19,14 +19,23 @@ function ProductList() {
+    fetchProducts();
   };
 
   const handleAdd = async (newProduct) => {
     // TODO: POST API를 호출하고 fetchProducts() 호출
+    const response = await axios.post(API_URL, newProduct);
+    fetchProducts();
   };
 
   const handleEdit = async (updatedProduct) => {
     // TODO: PUT API를 호출하고 fetchProducts() 호출
+    const response = await axios.put(
```

ESLint

[error] 'response' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)